src/services/types.d.ts

```diff
@@ -0,0 +1,23 @@
+import { Config } from '../types';
+
+/**
+ * Interface for sending data to different platforms
+ */
+export interface InterfaceAPI {
+  /**
+   * Client config
+   */
+  configuration?: Config;
+  /**
+   * Client initialization
+   *
+   * @param {Config} configuration - Client configuration.
+   */
+  init(configuration: Config): void;
```

Comment:
Can we provide a configuration to constructor args and initialise a connection in the constructor?